src/gui/mzroll/tabledockwidget.cpp

```diff
@@ -556,7 +556,10 @@ void TableDockWidget::noPeakFound()
 {
   QMessageBox *warning = new QMessageBox(this);
 
-    auto htmlText = QString("<p>No peaks found.</p>");
+    auto htmlText = QString("<p>No peaks we detected. \
```

Comment:
"No peaks _were_ detected...". My mistake in last comment. You pasted the mistake too :)

Reply:
oh.. ;p